modules/ding_event/ding_event.pages_default.inc

```diff
@@ -144,6 +144,8 @@ function ding_event_default_page_manager_handlers() {
   $display->cache = array();
   $display->title = '%node:title';
   $display->uuid = '45175b89-df7a-48fe-8118-cb87907dc9df';
+  $display->storage_type = 'page_manager';
```

Comment:
How is this related to the case?

Reply:
This is required by panels module in order to fix some access permissions issues, introduced in version 7.x-3.6, when on pane editing administrators had "access denied" error.